db/migrate/20230410121228_update_assessment_table.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateAssessmentTable < ActiveRecord::Migration[6.0]
```

Comment:
Would this migration be more aptly named: `AddAllowRecordDraftAnswerToAssessments` in accordance to Rails?